components/user-claim-retriever/src/main/java/com/wso2telco/claims/RemoteClaimsRetriever.java

```diff
@@ -110,7 +113,7 @@ private String getclaimValue(Iterator<String> i, boolean isHashed, Map<String, O
         try {
             Class c = Class.forName(className);
             RemoteClaims remoteClaims = (RemoteClaims) c.newInstance();
```

Comment:
Rather than instantiating class per each method call, isn't it better to use a singleton and instantiate in the first call? (or else using a DataHolder pattern)

Reply:
fixed